pkg/server/sandbox_run_windows.go

```diff
@@ -165,11 +166,12 @@ func (c *criService) RunPodSandbox(ctx context.Context, r *runtime.RunPodSandbox
 	log.G(ctx).Debugf("Sandbox container %q spec: %#+v", id, spew.NewFormatter(spec))
 
 	sandboxLabels := buildLabels(config.Labels, containerKindSandbox)
+	snapshotterOpt := snapshots.WithLabels(config.Annotations)
```

Comment:
We need this for the container activation as well.

Reply:
The other thing is that knowing this is in upstream now: https://github.com/containerd/containerd/pull/3055/commits/75f183887a99de2fd29d3144e5f9a1e09a069a67#diff-491f07c28eb851b720f11ce0d3dbb4f8R38

We should make the label conform to that naming convention.